administrative-sdk/basicAuth.js

```diff
@@ -0,0 +1,96 @@
+/* eslint-disable
```

Comment:
I doubt if those eslint-disable docs apply to all classes.... Splitting this file is an improvement nonetheless, so I'll accept it for now.